parlai/core/metrics.py

```diff
@@ -391,12 +418,19 @@ def aggregate_named_reports(named_reports: Dict[str, Dict[str, Metric]]):
     :param reports: Dict of tasks -> metrics.
     """
     # reporters is a list of teachers or worlds
-    m: Dict[str, Metric] = {}
+    m: Dict[str, Union[Metric, Dict[str, Metric]]] = {}
```

Comment:
So this allows nested dicts? I thought you didn't want that, right? And how below would nested dicts occur, since it looks like `task_report[each_metric]` needs to be a `Metric` in order to have an `.is_global()` method?

Reply:
ah actually no, no nested dicts.